OpenTokSamples/app/build.gradle

```diff
@@ -18,7 +18,7 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.opentok.android:opentok-android-sdk:2.8.+'
```

Comment:
This is unnecessary, it should always pull the latest 2.8 version... aka 2.8.1, did it not?